cmd/podman/ioprojectatomicpodman/io.projectatomic.podman.varlink

```diff
@@ -17,6 +17,35 @@ type StringResponse (
     message: string
 )
 
+type ImageInList (
```

Comment:
Can we get comments on all of these structs in the .varlink? I understand you're working to document the generated Go, but we should also try and provide some docs for the API definition as well.

In particular - what is ImageInList? That name is really confusing

Reply:
I will add some comments to describe the structures, sure.